core/src/machine/ppu.rs

```diff
@@ -121,6 +121,36 @@ impl Ppu {
         }
     }
 
+    /// Loads a byte from OAM at the given (absolute!) address.
+    ///
+    /// The given address has to be in `0xFE00..0xFEA0`, otherwise this
+    /// function panics!
+    ///
+    /// This function behaves like the real OAM. Meaning: during pixel
+    /// transfer and OAM search, this returns garbage.
+    pub(crate) fn load_oam_byte(&self, addr: Word) -> Byte {
+        match self.phase() {
+            Phase::PixelTransfer => Byte::new(0xff),
+            Phase::OamSearch => Byte::new(0xff),
```

Comment:
Maybe rather `Phase::PixelTransfer | Phase::OamSearch => Byte::new(0xff)`?

Reply:
I thought about this too, but think the current solution is more explicit and better readable.

Reply:
I disagree :D I think it's just duplicate code (`Byte::new(0xff)`) in your version.

But I won't fight over this, so we can keep it ;-)